pkg/kev/override.go

```diff
@@ -314,6 +314,12 @@ func (o *composeOverride) mergeServicesInto(p *ComposeProject) error {
 
 		envVarsFromNilToBlankInService(base)
 
+		// Copy over image name if one has bee defined in the override. In
+		// future this may expand to invlude other fields.
+		if override.Image != "" && override.Image != base.Image {
```

Comment:
minor nitpick. It'd be better to trim whitespaces and check for length instead in this condition I think.